### ISSUE TITLE
Request/ sec

### ISSUE THREAD

**Issue by satishviswanathan** [fabiolb/fabio#800]
Is there a benchmarking done for request per second. If so can you please share them.

**Comment by ketzacoatl** [comment]
No, though anecdotally, it was originally run at ebay with relatively high load for long periods of time.

A benchmark would depend on the host resources, and configuration. A test is pretty easy to setup with a load generator such as locust. Run some tests and publish your results, that would be a great addition to the FAQ!

**Comment by marco-m** [comment]
@tristanmorgan looks like this one can be closed ?

**Comment by tristanmorgan** [comment]
> Results may vary

Sorry, performance of Fabio will depend on specific environments. There has been work done to ensure Fabio remains lightweight and performant. eg. #928
